stable/xray/values.yaml

```diff
@@ -111,12 +111,22 @@ mongodb:
 ##
 rabbitmq-ha:
   replicaCount: 1
-  rabbitmqUsername: "guest"
```

Comment:
I think the quotes were added for a reason, maybe this is not applicable to the new chart.

Reply:
Yes. Was already tested with `guest` and works well.